test/integration_test/basic_test.go

```diff
@@ -139,6 +141,29 @@ var testStorageClusterBasicCases = []types.TestCase{
 		}),
 		TestFunc: BasicInstallWithNodeAffinity,
 	},
+	{
+		TestName:        "BasicInstallWithPxSaTokenRefresh",
+		TestrailCaseIDs: []string{},
```

Comment:
What is this Case ID for? Couldn't figure out what ids I should put here

Reply:
We have to create a testcase in TestRail and use that ID.

Reply:
Any instruction on how I can do that?